Source/ACE.Server/WorldObjects/Player_Inventory.cs

```diff
@@ -544,6 +544,34 @@ public void HandleActionPutItemInContainer(ObjectGuid itemGuid, ObjectGuid conta
             // Is this something I already have? If not, it has to be a pickup - do the pickup and out.
             if (item == null)
             {
+                var itemToPickup = CurrentLandblock?.GetObject(itemGuid);
+
+                if (itemToPickup != null)
+                {
+                    //Checking to see if item to pick is an container itself
+                    if (itemToPickup.WeenieType == WeenieType.Container)
+                    {
+                        //Check to see if the container is open
+                        if (itemToPickup.IsOpen ?? false)
+                        {
+                            var containerToPickup = CurrentLandblock?.GetObject(itemGuid) as Container;
+
+                            if (containerToPickup.Viewer == Session.Player.Guid.Full)
+                            {
+                                //We're the one that has it open. Close it before picking it up
+                                containerToPickup.Close(Session.Player);
+                            }
+                            else
+                            {
+                                //We're not who has it open. Can't pick up something someone else is viewing!
+                                Session.Network.EnqueueSend(new GameEventWeenieErrorWithString(Session, WeenieErrorWithString.The_IsCurrentlyInUse, itemToPickup.Name));
```

Comment:
Can you add a note here that notes something along the lines of what you wrote in the initial PR: "The messages returned when a user attempts to place pack being viewed by another user were what I remembered of retail. I couldn't find anything in PCAPs to support what I remembered. "

Reply:
Thanks for the review @Mag-nus. My latest commit(s) address your suggestion. This should be good to merge now.